gravitee-apim-console-webui/src/management/api/notifications/notifications-list/notofication-details/notification-details.component.scss

```diff
@@ -0,0 +1,32 @@
+@use 'sass:map';
+
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+@use '../../../../../scss/gio-layout' as gio-layout;
+
+:host {
+  @include gio-layout.gio-responsive-content-container;
+}
+
+.checkbox-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.mat-checkbox {
```

Comment:
What do you think about adding dedicated class and not using the material classes? 
And get closer to the BEM convention. 
We should avoid using them directly most of the time

Reply:
Fixed